cmake/modules/CompilerFlags.cmake

```diff
@@ -54,7 +54,7 @@ if(NOT MSVC)
 else() # MSVC
 	set(MINIMAL_BUILD ON)
 
-	set(FALCOSECURITY_LIBS_COMMON_FLAGS "-D_CRT_SECURE_NO_WARNINGS -DWIN32 -DMINIMAL_BUILD /EHsc /W3 /Zi /std:c++17")
+	set(FALCOSECURITY_LIBS_COMMON_FLAGS "-D_CRT_SECURE_NO_WARNINGS -DWIN32 -DMINIMAL_BUILD /EHsc /W3 /Zi /std:c++20")
```

Comment:
Why C++20?

Reply:
The msvc compiler complains on designated initializers on c++17 (afaik gcc supports it even if is not in spec). So there are 2 ways:
- change the standard to c++20 without applying any other change (as done in this pr) or
- remove the designated initializers (i.e. from `.fdinfo_count = m_fdinfos.back().size()` to `/* .fdinfo_count = */ static_cast<int>(m_fdinfos.back().size())`)

wdyt?